src/packagers/Packager.js

```diff
@@ -17,6 +17,7 @@ class Packager {
 
   async start() {}
 
+  // eslint-disable-next-line no-unused-vars
```

Comment:
Why is this particular one needed? I don't see any unused vars.

Reply:
`asset` is unused. Guessing it was left to be self-documenting since this requires implementation by extending classes.

Reply:
Exactly: I prefered not to remove it, because it seems to be useful for documentation